src/components/Plugin/Plugin.tsx

```diff
@@ -136,6 +141,53 @@ export function PluginStore(props: PluginProps) {
       })
     : null;
 
+  const clonedTopActionDisabled = preIcon
+    ? cloneElement(topAction, {
+        className: 'w-6 h-6 text-tertiary cursor-pointer',
+        // eslint-disable-next-line @typescript-eslint/no-empty-function
+        onClick: () => {},
+      })
+    : null;
+  const clonedWarning = cloneElement(
+    <FiAlertTriangle {...topAction} color="orange" />,
```

Comment:
for the color you should use on of the massa preset colors.... or the exact hex value of massa warning